crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -555,36 +555,35 @@ en:
       header: 'What do you want to do?'
       install: 'Install from scratch'
       restore: 'Restore from backup'
-      upgrade: 'Upgrade from backup'
+      upgrade: 'Continue Upgrade from SUSE OpenStack Cloud 5'
     upgrades:
       title: 'Upgrade'
-      header: '%{product}: Upgrade from backup'
+      header: '%{product}: Upgrade from SUSE OpenStack Cloud 5'
       nav:
-        prepare: 'Prepare upgrade'
-        download: 'Data download'
-        confirm: 'Confirm upgrade'
-        start: 'Start upgrade'
-        restore: 'Restore Crowbar'
-        repos: 'Repositories'
-        services: 'Stop services'
-        backup: 'Data backup'
-        nodes: 'Upgrading nodes'
-        finish: 'Upgrade finished'
+        prepare: 'Prepare Client Nodes'
+        download: 'Download Upgrade Data'
+        confirm: 'Reinstall Admin Server'
+        start: 'Continue Upgrade'
+        restore: 'Import Upgrade Data'
+        repos: 'Verify Repositories'
+        services: 'Stop OpenStack Services'
+        backup: 'Data Backup'
+        nodes: 'Upgrading Nodes'
+        finish: 'Upgrade Finished'
       start:
         cancel: 'Back to selection'
-        continue: 'Continue upgrade process'
-        header: 'Start upgrade'
+        continue: 'Upload and continue'
+        header: 'Continue Upgrade'
         description: |
-          Lorem Ipsum is simply dummy text of the printing and typesetting
-          industry. Lorem Ipsum has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a galley of type and
-          scrambled it to make a type specimen book.
+          Select the location of the upgrade data archive that you created old Administration server
+          using where you started the upgrade wizard. Upload it by clicking 'Upload and continue'.
       restore:
         continue: 'Continue upgrade process'
-        header: 'Restore Crowbar'
-        install_button: 'Start Installer'
+        header: 'Import Upgrade Data'
```

Comment:
here we call it import upgrade data and below, "restore". why not let the header like it was before? it is a bit confusing as subsequently we always use the term "restore"